runtime/doc/diagnostic.txt

```diff
@@ -590,6 +600,12 @@ Lua module: vim.diagnostic                                    *diagnostic-api*
                      diagnostics matching the given severity
                      |diagnostic-severity|.
 
+*vim.diagnostic.Opts.VirtualLines*
+
+    Fields: ~
+      • {only_current_line}?  (`boolean`, default: `true`) Only show
```

Comment:
```suggestion
      • {current_line}?  (`boolean`, default: `true`) Only show
```

Reply:
AFAICT there are no other `only_` fields. I doubt this is the first time we have needed such a boolean. And it's true that names can have minor ambiguities without knowing the type--but the type is available to resolve those minor ambiguities when they arise. That's a good tradeoff to avoid over-explaining names in general.

Reply:
Fair enough. I'll rename it.